tgui/packages/tgui/interfaces/Biogenerator.jsx

```diff
@@ -116,7 +116,7 @@ const Products = (props) => {
   const { act, data } = useBackend();
   const { biomass, product_list } = data;
 
-  let [vendAmount, setVendAmount] = useSharedState('vendAmount', 1);
+  let [vendAmount, setVendAmount] = useShareState('vendAmount', 1);
```

Comment:
Bro changed `useSharedState` to `useSharedState`

Reply:
Watch me commit sin.

```suggestion
  let [vendAmount, setVendAmount] = useSharedState('vendAmount', 1);
```

Reply:
> Watch me commit sin.

Oh, missed `d` huh